scripts/check-go-version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
```

Comment:
I suggest to expand the flags to `set -euo pipefail`. It ensure exiting immediately also on right-most pipe fails and unset variables/parameters.